langlib/lang.stream/src/main/ballerina/stream.bal

```diff
@@ -149,7 +187,12 @@ public isolated function iterator(stream<Type,CompletionType> stm) returns objec
 # Closes a stream.
 #
 # This releases any system resources being used by the stream.
-# Closing a stream that has already been closed has no efffect and returns `()`.
+# Closing a stream that has already been closed has no effect and returns `()`.
+#
+# ```ballerina
+# stream<int> scores = [45, 60, 75, 30, 90].toStream();
+# _ = scores.close();
```

Comment:
Shall we add another sample with a completion other than (). wdyt?
```ballerina
function name(stream<int, error?> strm) {
    error? r = strm.close();
}
```

Reply:
@pcnfernando do you have a compact example to show this (instead of above)?

Reply:
No, if we are to create a steeam with other completions it will be much lengthier. Hence, this would be the most compact version as I see it

Reply:
ya, guessed it. I don't prefer above example though 😄

Reply:
How about

```
stream<int, error?> strm = new;
error? res = strm.close();
```

although new doesn't really help.

Generally, the completion type will include error right? So may be helpful to not emphasize (just) ignoring with `_`.

Reply:
IMO, I don't think this really helps. But I am OK to add this since we wanted to show the error completion type. Will change it then.

Reply:
I'll go with this,

```
    stream<int, error?> strm = new;
    _ = check strm.close();
```

Reply:
We don't need to ignore (). Hence, shall we do either
```ballerina
    stream<int, error?> strm = new;
    check strm.close();
```
or
```ballerina
    stream<int, error?> strm = new;
    error? res = strm.close();
```

Reply:
Let's go with 1st option then.